components/CreateOrganizationForm.js

```diff
@@ -1,289 +1,421 @@
-            section#location {
-              margin-top: 0;
-            }
+    if (!values.website.match(new RegExp(regexExp)) || values.website.startsWith('http')) {
+      errors.website = intl.formatMessage(orgMessages.errorWebsite);
+    }
+    return errors;
+  };
+  const submit = values => {
+    const { name, slug, description, website } = values;
+    onSubmit({ name, slug, description, website, authorization });
+  };
 
-            .image .InputTypeDropzone {
-              width: 100px;
-            }
+  const removeAdmin = collective => {
+    const filteredAdmins = admins.filter(admin => admin.member.id !== collective.id);
+    setAdmins(filteredAdmins);
+    updateAdmins(admins);
+  };
 
-            .backgroundImage-dropzone {
-              max-width: 500px;
-              overflow: hidden;
-            }
+  // Update admins whenever there is a change
+  useEffect(() => {
+    if (admins.length) {
+      updateAdmins(admins);
+    }
+  }, [admins]);
 
-            .user .image-dropzone {
-              width: 64px;
-              height: 64px;
-              border-radius: 50%;
-              overflow: hidden;
+  return (
+    <Flex flexDirection="column">
+      <Flex flexDirection="column" my={[0, 2]}>
+        <Box>
+          <BackButton asLink onClick={() => window && window.history.back()}>
+            ←&nbsp;
+            <FormattedMessage id="Back" defaultMessage="Back" />
+          </BackButton>
+        </Box>
+        <Box mb={[1, 3]} alignItems="center">
+          <H1
+            width={[288, 264]}
+            fontSize="28px"
+            lineHeight="36px"
+            fontWeight="500"
+            textAlign={['left', 'left', 'left', 'center']}
+            color="black.900"
+            letterSpacing="1px"
+            mt={2}
+            px={[2, 4, 0]}
+          >
+            <FormattedMessage id="create.org.title" defaultMessage="Create Organization" />
+          </H1>
+        </Box>
+      </Flex>
+      {error && (
+        <Flex alignItems="center" justifyContent="center">
+          <MessageBox type="error" withIcon mb={[1, 3]} data-cy="cof-error-message">
+            {error}
+          </MessageBox>
+        </Flex>
+      )}
+      <Formik validate={validate} initialValues={initialValues} onSubmit={submit} validateOnChange={true}>
+        {formik => {
+          const { values, handleSubmit, errors, touched, setFieldValue } = formik;
+          const suggestedSlug = value => {
+            const slugOptions = {
+              replacement: '-',
+              lower: true,
+              strict: true,
+            };
+            return trim(slugify(value, slugOptions), '-');
+          };
+          const handleSlugChange = e => {
+            if (!touched.slug) {
+              setFieldValue('slug', suggestedSlug(e.target.value));
             }
+          };
+          return (
+            <Form>
+              <Container flexDirection="column" justifyContent="center" px={[1, 30, 150]} mt={4}>
```

Comment:
Right now the whole page is not centered anymore on widescreen/desktop/tablet:

![Screenshot 2020-12-14 at 12 25 58](https://user-images.githubusercontent.com/37520401/102082090-3fbe5c80-3e09-11eb-9088-39aeaf080abf.png)

And on mobile it's also on the left and not centered anymore. I'm not sure what changed between the last version of the PR and here but everything should have the same spacing/centering as before.

![Screenshot 2020-12-14 at 12 39 17](https://user-images.githubusercontent.com/37520401/102082176-654b6600-3e09-11eb-90dc-c3cb1fa7977b.png)

Compare with the Figma where things have more space around them and are centered on mobile:

![Screenshot 2020-12-14 at 12 40 02](https://user-images.githubusercontent.com/37520401/102082255-890eac00-3e09-11eb-93b2-6dc35efe7fa8.png)

So double check the centering and spacing for things.

Reply:
This is strange 😞

Reply:
fixed